signer/src/testing/dummy.rs

```diff
@@ -323,23 +321,23 @@ impl fake::Dummy<fake::Faker> for WithdrawalRejectEvent {
     fn dummy_with_rng<R: Rng + ?Sized>(config: &fake::Faker, rng: &mut R) -> Self {
         let bitmap = rng.next_u64() as u128;
         WithdrawalRejectEvent {
-            txid: blockstack_lib::burnchains::Txid(config.fake_with_rng(rng)),
-            block_id: stacks_common::types::chainstate::StacksBlockId(config.fake_with_rng(rng)),
+            txid: config.fake_with_rng(rng),
+            block_id: config.fake_with_rng(rng),
             request_id: rng.next_u32() as u64,
             signer_bitmap: BitArray::new(bitmap.to_le_bytes()),
         }
     }
 }
 
-impl fake::Dummy<fake::Faker> for WithdrawalCreateEvent {
+impl fake::Dummy<fake::Faker> for crate::storage::model::WithdrawalCreateEvent {
```

Comment:
We've already imported `crate::storage::model::WithdrawalCreateEvent` here so no need for the qualifier.